capacity-mode-evaluator/src/pricing.py

```diff
@@ -5,7 +5,7 @@
 
 
 class PricingUtility(object):
-    def __init__(self, region_name):
+    def __init__(self, region_name, profile_name='default'):
```

Comment:
This change will force everyone to use a profile, there are customers that use this tool with environment variables.

Reply:
Updated to `None`